src/Resources/views/form_theme.html.twig

```diff
@@ -38,9 +38,10 @@
 {% block zicht_quicklist_autocomplete_single_widget %}
     {{ block('zicht_quicklist_autocomplete_control') }}
 
-    {% if not required %}
-        <input type="checkbox" class="null-control" {% if not value.id %}checked{% endif %}>
-    {% endif %}
+    {# Unsure what this is for. TODO remove in next major version #}
+{#    {% if not required %}#}
+{#        <input type="checkbox" class="null-control" {% if not value.id %}checked{% endif %}>#}
+{#    {% endif %}#}
```

Comment:
Als het "TODO remove in next major version" is, moet je het dan niet juist laten staan (niet uit-commenten). En in de next major versie weghalen?

Reply:
Ik wil het wel graag weg hebben, het ziet er raar uit en lijkt echt niks te doen en volgens mij hebben we dit in de andere weergaves al eerder verwijderd.

Reply:
Dan zou ik het gewoon geheel verwijderen